.travis.yml

```diff
@@ -2,6 +2,8 @@ language: rust
 env:
   global:
     - secure: g4kCg8twONwKPquuJmYrvGjo2n0lNtWTbyzFOITNn8FgCxNK2j38Qc9/UhErTR3g3rDjVzsTHZ8FTH7TJZrOK1Nzz90tJG6JHqUv77ufkcBlxgwwjilOz84uQhkDTMpLitMEeQDLEynKeWbxrjtc5LIpjEkxOPk5eiqwzKRN14c=
+before_script:
+  - mkdir -p .cargo && echo 'paths = ["python27-sys"]' > .cargo/config
```

Comment:
yes we could, I just wasn't sure whether you had that in mind and I wonder if it's impolite to take precedence over any configs further up the dir tree..? precedence rules aren't made very clear here:

http://doc.crates.io/config.html